packages/components/src/FocusTrap.test.tsx

```diff
@@ -11,7 +9,11 @@ describe('FocusTrap', () => {
     const user = userEvent.setup();
 
     const { container } = render(
-      h(Fragment, [h(FocusTrap, [button(['A']), button(['B']), button(['C'])]), button(['D'])])
+      <FocusTrap onEscape={jest.fn()}>
```

Comment:
In a plus for JSX vs our own React Hyperscript Helpers types, JSX identified that this was missing a required prop.

Reply:
love this... we have been missing out on a lot of stuff like this...